lib/parser/openmp-grammar.h

```diff
@@ -97,18 +104,19 @@ TYPE_PARSER(construct<OmpScheduleClause>(maybe(Parser<OmpScheduleModifier>{}),
 
 // 2.12 IF (directive-name-modifier: scalar-logical-expr)
 TYPE_PARSER(construct<OmpIfClause>(
-    maybe(("PARALLEL" >> pure(OmpIfClause::DirectiveNameModifier::Parallel) ||
-              "TARGET ENTER DATA" >>
-                  pure(OmpIfClause::DirectiveNameModifier::TargetEnterData) ||
-              "TARGET EXIT DATA" >>
-                  pure(OmpIfClause::DirectiveNameModifier::TargetExitData) ||
-              "TARGET DATA" >>
-                  pure(OmpIfClause::DirectiveNameModifier::TargetData) ||
-              "TARGET UPDATE" >>
-                  pure(OmpIfClause::DirectiveNameModifier::TargetUpdate) ||
-              "TARGET" >> pure(OmpIfClause::DirectiveNameModifier::Target) ||
-              "TASK"_id >> pure(OmpIfClause::DirectiveNameModifier::Task) ||
-              "TASKLOOP" >> pure(OmpIfClause::DirectiveNameModifier::Taskloop)) /
+    maybe(
+        ("PARALLEL" >> pure(OmpIfClause::DirectiveNameModifier::Parallel) ||
+            "TARGET ENTER DATA" >>
+                pure(OmpIfClause::DirectiveNameModifier::TargetEnterData) ||
+            "TARGET EXIT DATA" >>
+                pure(OmpIfClause::DirectiveNameModifier::TargetExitData) ||
+            "TARGET DATA" >>
+                pure(OmpIfClause::DirectiveNameModifier::TargetData) ||
+            "TARGET UPDATE" >>
+                pure(OmpIfClause::DirectiveNameModifier::TargetUpdate) ||
+            "TARGET" >> pure(OmpIfClause::DirectiveNameModifier::Target) ||
+            "TASK"_id >> pure(OmpIfClause::DirectiveNameModifier::Task) ||
+            "TASKLOOP" >> pure(OmpIfClause::DirectiveNameModifier::Taskloop)) /
```

Comment:
This seems to be an unrelated formatting change? Although the new formatting is what I get if I run clang-format on master too so it makes sense to keep it.

Reply:
Yes, it's unrelated formatting change by clang-format.